WalletWasabi.Gui/Behaviors/CommandOnDoubleClickBehavior.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Avalonia.Controls;
+using System.Reactive.Disposables;
+
+namespace WalletWasabi.Gui.Behaviors
+{
+	public class CommandOnDoubleClickBehavior : CommandBasedBehavior<Control>
+	{
+		private CompositeDisposable Disposables { get; set; }
+
+		protected override void OnAttached()
+		{
+			Disposables = new CompositeDisposable();
+
+			base.OnAttached();
+
+			Disposables.Add(AssociatedObject.AddHandler(Control.PointerPressedEvent, (sender, e) =>
+			{
+				if (e.ClickCount == 2)
```

Comment:
use the DoubleTapped event... https://github.com/AvaloniaUI/Avalonia/blob/9e6b878bce8e1effa29f85c939927f5786f75fa7/src/Avalonia.Input/InputElement.cs#L147